packages/xrpl/test/integration/utils.ts

```diff
@@ -10,13 +10,21 @@ import {
   type SubmitResponse,
   TimeoutError,
   NotConnectedError,
+  Currency,
 } from '../../src'
 import { Payment, Transaction } from '../../src/models/transactions'
 import { hashSignedTx } from '../../src/utils/hashes'
 
 const masterAccount = 'rHb9CJAWyB4rj91VRWn96DkukG4bwdtyTh'
 const masterSecret = 'snoPBrXtMeMyMHUVTgbuqAfg1SUTb'
 
+export interface SetupAMMPool {
```

Comment:
IMO this should just be `AMMPool` - I saw the import in `setup.ts` and thought you had mis-capitalized a function.